src/selectors.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {selectorMatchesAnyScope, matcherForSelector}
+
+const _ = require('underscore-plus')
+
+/**
+ * Parse a selector into parts. If already parsed, returns the selector
+ * unmodified.
+ * 
+ * @param {String|Array<String>} selector
+ * @returns {Array<String>} selector parts
+ */
```

Comment:
There's a slightly different (and more unusual 😬) style of doc comments called [AtomDoc](https://github.com/atom/atomdoc) that we usually use in this codebase. There's good examples of it in [this file](https://github.com/atom/atom/blob/master/src/text-editor.js#L2565-L2576). I wouldn't necessarily be opposed to switching to JSDoc at some point. In the meantime, could adjust these to use atomdoc?

Reply:
👍